lib/bundles.cf

```diff
@@ -439,21 +439,29 @@ bundle agent collect_vars(name, tag, flatten)
       "todo_$(name)"    slist => variablesmatching(".*", $(tag));
 
     !flatten::
-      "$(name)" data => parsejson('{}'),            policy => "free";
+      "$(name)"
+        data => parsejson('{}'),
+        policy => "free";
 
       # this iterates!
-      "$(name)" data => mergedata($(name), '{ "$(todo_$(name))": $(todo_$(name)) }'),
+      "$(name)"
+        data => mergedata($(name), '{ "$(todo_$(name))": $(todo_$(name)) }'),
```

Comment:
Heh, I thought it read cleaner this way. I never use policy => free anymore since vars have been free by default since 3.3 era